middleware/assignLabels.js

```diff
@@ -1,6 +1,6 @@
 function setup(labelGenerator) {
   function middleware(req, res, next) {
-    return assignLabel(req, res, next, labelGenerator);
+    return assignLabel(req, res, next, labelGenerator || require('pelias-labels'));
```

Comment:
Even though I think this setup function will only be called once, so this won't cause `pelias-labels` to be required too many times, it's probably less confusing to keep the require statement at the start of this file.